### DISCUSSION TITLE
Give a review on "html 2 react: The Ultimate guide" book

### DISCUSSION THREAD

**Discussion by ghost** [EddieHubCommunity/support#149] [Q&A] [answered]
Link: https://gumroad.com/l/html2react

**Comment by eddiejaoude** [comment, marked as answer]
Its difficult to give a review on something I am not familiar with and have used / read. But there are many reviews (75 people) that give it a high rating, and it was published in `June 2020`, so that looks promising. 

Did you look at the sample available? If everyone the authors style is sometimes better / worse for different people, see if you can get on with it https://drive.google.com/file/d/1jIuue1bL9h9uUD7SMhZyGcs-vb-8wY2w/view

<img width="373" alt="Screenshot 2020-09-07 at 10 56 27" src="https://user-images.githubusercontent.com/624760/92369061-cff95100-f0f8-11ea-8859-5883403b32f9.png">

**Reply by ghost** [reply]
This helps a lot @eddiejaoude. Thanks!

**Reply by eddiejaoude** [reply]
You are welcome @JoshuaPoddoku

**Reply by ghost** [reply]
Thanks man
